packages/ui/src/components/home/sections/PostsSection.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+
+import {
+    CreatePostInput
+} from '../CreatePostInput';
+import { Post } from "../../post/Post";
+
+import StyledMarkdown from '../../../../../web/src/markdown/StyledMarkdown';
```

Comment:
Don't import things like this from other package files, add it as props instead.